### PR TITLE
Rae moar development

### PR DESCRIPTION
**Pull request by rmr167**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/Account.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** An item on a todo list. */
```
Comment (geoffreyzgoog) [inline portfolio/src/main/java/com/google/sps/data/Account.java]: Update javadoc.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -24,9 +39,78 @@
...
+
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        PreparedQuery results = datastore.prepare(query);
+
+        List<Task> tasks = new ArrayList<>();
+        int i = 0;
+        for (Entity entity : results.asIterable()) {
+            long id = entity.getKey().getId();
+            String text = (String) entity.getProperty("text");
+            long timestamp = (long) entity.getProperty("timestamp");
```
Comment (geoffreyzgoog) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:59]: Does the cast work?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -24,9 +39,78 @@
...
+
+    private String getUserNickname(String id) {
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        Query query =
+            new Query("UserInfo")
+                .setFilter(new Query.FilterPredicate("id", Query.FilterOperator.EQUAL, id));
+        PreparedQuery results = datastore.prepare(query);
+        Entity entity = results.asSingleEntity();
+        if (entity == null) {
+        return "";
```
Comment (geoffreyzgoog) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Indent.

Comment (rmr167) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Thanks! Indent added.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (rmr167) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:59]: I believe so. Were you referring to the timestamp, text, or id?

Comment (geoffreyzgoog) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:59]: Timestamp. I thought Entity properties are strings but now I see it's actually Object.

**Review by geoffreyzgoog — APPROVED** [review]

**Pull request merged**